main.go

```diff
@@ -19,18 +18,17 @@ func main() {
 
 	ctx, cancel := context.WithCancel(context.Background())
 
-	config, err := config.Read()
-	if err != nil {
-		panic(err)
-	}
+	config_otel_endpoint := os.Getenv("OTEL_ENDPOINT")
```

Comment:
Typically in Golang the convention for naming variables is to use ~~camelCase~~ mixed caps
https://go.dev/doc/effective_go#mixed-caps

Reply:
✅